.github/workflows/delete-review-app.yml

```diff
@@ -25,83 +25,14 @@ jobs:
-          echo "::add-mask::$TFSTATE_CONTAINER_ACCESS_KEY"
-          echo "TFSTATE_CONTAINER_ACCESS_KEY=$TFSTATE_CONTAINER_ACCESS_KEY" >> $GITHUB_ENV
-        shell: bash
-
-      - uses: hashicorp/setup-terraform@v3
-        with:
-          terraform_version: 1.6.4
-          terraform_wrapper: false
-
-      - name: Set Environment variables
-        run: |
-          TF_STATE_FILE=pr-${{ env.PR_NUMBER }}_kubernetes.tfstate
-          echo "TF_STATE_FILE=$TF_STATE_FILE" >> $GITHUB_ENV
-
-          pr_state_file=$(az storage blob list -c terraform-state \
-            --account-key ${{ env.TFSTATE_CONTAINER_ACCESS_KEY }} \
-            --account-name ${{ env.STORAGE_ACCOUNT_NAME }} \
-            --prefix $TF_STATE_FILE --query "[].name" -o tsv)
-          if [ -n "$pr_state_file" ]; then
-            echo "TF_STATE_EXISTS=true" >> $GITHUB_ENV
-          fi
-
-      - uses: DFE-Digital/github-actions/set-kubelogin-environment@master
+      - uses: DFE-Digital/github-actions/delete-review-app@master
         with:
           azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
           azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
           azure-subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-
-      - uses: google-github-actions/auth@v2
-        with:
-          project_id: teaching-qualifications
-          workload_identity_provider: projects/708780292301/locations/global/workloadIdentityPools/access-your-teaching-qualificati/providers/access-your-teaching-qualificati
-
-      - name: Terraform Destroy
-        run: |
-          make ci review terraform-destroy
-        env:
-          PR_NUMBER: ${{ env.PR_NUMBER }}
-
-      - name: Delete tf state file
-        if: env.TF_STATE_EXISTS == 'true'
-        run: |
-          az storage blob delete -c terraform-state --name ${{ env.TF_STATE_FILE }} \
-          --account-key ${{ env.TFSTATE_CONTAINER_ACCESS_KEY }} \
-          --account-name ${{ env.STORAGE_ACCOUNT_NAME }}
-
-      - name: Post Pull Request Comment ${{ github.event.number }}
-        if: ${{ github.event_name == 'pull_request' }}
-        uses: marocchino/sticky-pull-request-comment@v2
-        with:
-          header: aks
-          message: |
-            The review apps Access Your Teaching Qualifications & Check A Teacher's Record have been deleted.
-            The following domains are not available anymore:
-            - <https://access-your-teaching-qualifications-pr-${{ github.event.number }}.test.teacherservices.cloud>
-            - <https://check-a-teachers-record-pr-${{ github.event.number }}.test.teacherservices.cloud>
+          pr-number: ${{ github.event.pull_request.number || github.event.inputs.pr_number }}
+          resource-group-name: s189t01-aytq-rv-rg
+          storage-account-name: s189t01aytqrvtfsa
+          tf-state-file: pr-${{ github.event.pull_request.number || github.event.inputs.pr_number }}/terraform.tfstate
```

Comment:
tf state file will be of the form pr-nnn_kubernetes.tfstate, this is setting pr-nnn/terraform.tfstate